docs/content/en/docs/getting-started/ports/_index.md

```diff
@@ -107,7 +134,7 @@ In addition to the [Ports Required to Use AWS Services on an AWS Snowball Edge D
 | TCP      | Inbound   | 2379        | Manager                 | Etcd API endpoint                 |
 | TCP      | Inbound   | 2380        | Manager                 | Etcd API endpoint                 |
 
-## Control plane management tools
+##### Control plane management tools
```

Comment:
```suggestion
### Control plane management tools
```
I suggested changing the heading above to ##. If you agree, then this should be ###.